docs/source/markdown/options/volume.md

```diff
@@ -156,6 +156,12 @@ To recursively mount a volume and all of its submounts into a
 used, and submounts of the source directory will not be mounted into the
 <<container|pod>>.
 
+Mounting the volume with the **copy** options tells podman to copy content from
+the underlying destination directory onto newly created internal volumes. This
+**copy** only happens on the initial creation of the volume and will not copy up
+content when the volume is used on different containers. The **copy** option
+is ignored on bind mounts ,`/HOST-DIR`, and has no effect.
```

Comment:
The last sentence does not parse. Did you mean something like this?
>has no effect on bind mounts or `/HOST-DIR`

Reply:
Above, we describe the bind mounts based on /HOST-DIR.

```
   --volume, -v=[[SOURCE-VOLUME|HOST-DIR:]CONTAINER-DIR[:OPTIONS]]
       Create  a bind mount. If -v /HOST-DIR:/CONTAINER-DIR is specified, Podman bind mounts /HOST-DIR from the host into /CONTAINER-DIR in the Podman container. Similarly, -v SOURCE-VOLUME:/CONTAINER-DIR will mount the named volume
       from the host into the container. If no such named volume exists, Podman will create one. If no source is given, the volume will be created as an anonymously named volume with a randomly generated name, and  will  be  removed
       when the container is removed via the --rm flag or the podman rm --volumes command.
```

So I am trying to bring users attention back to this.

Reply:
Removed since it caused confusion, and adds little value.